.gitignore

```diff
@@ -16,6 +16,7 @@ compile_commands.json
 /lib/spec/spec_helper.rb
 /debug*/
 /*release*/
+/*build*/
```

Comment:
Actually the project template already has it, but I'm going to change the way they match, for ease of use on VMs using shared folders, where you might need multiple build directories in one project.